src/app/app.component.ts

```diff
@@ -3,10 +3,9 @@ import {RouterOutlet} from '@angular/router';
 import {SnowflakesComponent} from './feature/calendar/core/snowflakes/snowflakes.component';
 
 @Component({
-  selector: 'xmas-root',
-  standalone: true,
-  imports: [RouterOutlet, SnowflakesComponent],
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css'],
+    selector: 'xmas-root',
+    imports: [RouterOutlet, SnowflakesComponent],
```

Comment:
wondering why standalone was removed?

Reply:
here and in other components

Reply:
standalone is removed in v19, it's the default mode

Reply:
Angular 19 = standalone default :)

Reply:
oh @aboudard beat me to it :D

Reply:
Yeah, I do that, I beat people 🦈

Reply:
damn :D

Reply:
What about pipes? were left with this attribute. should we remove standalone property from pipes too - just to make it cleaner and consistent?

Reply:
e.g. SafePipe, SocialProfileUrlPipe